src/quo2/components/buttons/button.cljs

```diff
@@ -66,7 +66,12 @@
                               :label                {:style    {:color colors/white}}
                               :background-color     {:default  colors/neutral-95
                                                      :pressed  colors/neutral-95
-                                                     :disabled colors/neutral-95}}}
+                                                     :disabled colors/neutral-95}}
+           :purple    {:icon-color       colors/white
+                       :label            {:style {:color colors/white}}
+                       :background-color {:default  (get-in colors/customization [:purple 50])
+                                          :pressed  (get-in colors/customization [:purple 50])
```

Comment:
I think you should use `custom-color` function here instead.
It's in the colors file.
e.g
```
(colors/custom-color :purple 50)
```

Reply:
Thanks! :) Didn't see that function